src/test/scala/nl.knaw.dans.easy.s2d/EasyToDataverseMapperSpec.scala

```diff
@@ -21,8 +21,8 @@ import org.scalatest.{ FlatSpec, Matchers, OneInstancePerTest }
 
 class EasyToDataverseMapperSpec extends FlatSpec with OneInstancePerTest with Matchers {
 
-  implicit val format = DefaultFormats
-  val mapper = new EasyToDataverseMapper
+  private implicit val format: DefaultFormats.type = DefaultFormats
+  private val mapper = EasyToDataverseMapper()
```

Comment:
Just for fun I tried the following. It actually produced a lot of empty fields. Is that intentionally?

```scala
  "mapToJson" should "..." in {
    mapper.mapToJson(
       <ddm:DDM>
            <ddm:profile>
              <ddm:title>Agricultural Sciences</ddm:title>
            </ddm:profile>
         </ddm:DDM>
    ) shouldBe
      """"{
        |  "datasetVersion":{
        |    "metadataBlocks":{
        |      "citation":{
        |        "displayName":"Citation Metadata",
        |        "fields":[
        |          {
        |            "typeName":"title",
        |            "multiple":false,
        |            "typeClass":"primitive",
        |            "value":"Agricultural Sciences"
        |          }
        |        ]
        |      }
        |    }
        |  }
        |}"""".stripMargin
  }
```